app/services/analytics_events.rb

```diff
@@ -220,6 +225,11 @@ def backup_code_created(enabled_mfa_methods_count:, **extra)
     )
   end
 
+  # Tracks when the user visits the Backup Code Regenerate page.
+  def backup_code_regenerate_visit(**properties)
```

Comment:
Can we list out the specific properties we're expecting here, and document them in the method (YARD) documentation? So that it's clear what it's logging and can be included in the [handbook analytics events documentation](https://handbook.login.gov/articles/analytics-events.html).

```suggestion
  # @param [String] request_came_from the controller/action the request came from
  def backup_code_regenerate_visit(request_came_from:, **extra)
```